ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -1325,33 +1223,31 @@ def test_torch_characteristic_equation_solver(
         safety_factor_scale="log",
         small_abs_safety_factor=6,
     ).filter(
-        lambda x: "float16" not in x[0]
-        and "bfloat16" not in x[0]
-        and np.linalg.cond(x[1][0][:, :-1]) < 1 / sys.float_info.epsilon
-        and np.linalg.det(x[1][0][:, :-1]) != 0
-        and np.linalg.cond(x[1][0][:, -1].reshape(-1, 1)) < 1 / sys.float_info.epsilon
+        lambda x: np.linalg.cond(x[1][0][:, :-1]) < 1 / sys.float_info.epsilon
+                  and np.linalg.det(x[1][0][:, :-1]) != 0
+                  and np.linalg.cond(x[1][0][:, -1].reshape(-1, 1)) < 1 / sys.float_info.epsilon
     ),
-    check=st.booleans(),
```

Comment:
Even though there is no `check` argument, there is a `left` argument to be tested.